vizro-ai/docs/pages/user-guides/customize-vizro-ai.md

```diff
@@ -24,7 +24,8 @@ vizro_ai = VizroAI(model="<chosen model>")
     | API key           | `OPENAI_API_KEY`                      |
     | Base API URL      | `OPENAI_API_BASE`                     |
 
-    To use OpenAI with Vizro-AI, you must have an account with paid-for credits available. None of the free accounts will suffice. You can check [all available OpenAI models including pricing on their website](https://platform.openai.com/docs/models). This will also explain which version the below string acronyms currently point to.
+    To use OpenAI with Vizro-AI, you must have an account with paid-for credits available. None of the free accounts will suffice. You can check [all available OpenAI models including pricing on their website](https://platform.openai.com/docs/models). This will also explain which version the below string acronyms currently point to. Before usage, please review OpenAI's guidelines on risk mitigation to understand potential model limitations and best practices.
+    More details on responsible usage can be found on this [link](https://platform.openai.com/docs/models).
```

Comment:
```suggestion
    [See the OpenAI site for more details on responsible usage](https://platform.openai.com/docs/models).
```

Reply:
Should we not link to this: https://platform.openai.com/docs/guides/safety-best-practices. I have only browsed this, but it seems more relevant?